src/layouts/sidebar-sidecar/utils/_local_platform-docs-mdx/index.tsx

```diff
@@ -86,17 +45,6 @@ function makeImageElement({ noBorder }: { noBorder: ImageProps['noBorder'] }) {
 	)
 }
 
-/**
- * Note: the MdxListItem component used below overrides the custom 'li'
- * running in the @hashicorp/remark-plugins "typography" plugin.
- *
- * This plugin is run via the DocsPage component in @hashicorp/docs-page
- * so we need to override it here. https://github.com/hashicorp/web-platform-packages/blob/main/packages/remark-plugins/plugins/typography/index.js#L13
- *
- * @TODO cleanup use of remark plugins in @hashicorp/docs-page so these overrides aren't necessary.
- * Or decouple the dev portal docs page from this legacy component. https://app.asana.com/0/1202097197789424/1202310153805071
- */
```

Comment:
Removed this `TODO` comment as I realized it's outdated - we've removed the `typography` anchor links plugin in #559. Added notes to and closed this task: https://app.asana.com/0/1202097197789424/1202310153805071/f